README.md

```diff
@@ -158,23 +160,97 @@ is different. See each providers `help` command for the available actions.
 Note, when uploading to some cloud providers (e.g. gce) the image may need to be packaged
 with a different --format (e.g. --format=gce) when running `image_to_vm.sh`
 
-## plume
+### plume
 Plume is the Container Linux release utility. Releases are done in two stages,
 each with their own command: pre-release and release. Both of these commands are idempotent.
 
-### plume pre-release
+#### plume pre-release
 The pre-release command does as much of the release process as possible without making anything public.
 This includes uploading images to cloud providers (except those like gce which don't allow us to upload
 images without making them public).
 
-## plume release
+### plume release
 Publish a new Container Linux release. This makes the images uploaded by pre-release public and uploads
 images that pre-release could not. It copies the release artifacts to public storage buckets and updates
 the directory index.
 
-### plume index
+#### plume index
 Generate and upload index.html objects to turn a Google Cloud Storage
 bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Platform Credentials
+Each platform reads the credentials it uses from different files.
+
+### aws
+Both Mantle and the `aws` command use the `~/.aws/credentials` file to store credentials.
+It can be generated using the `aws` command: 
+```
+$ aws configure
+```
+
+or be populated manually:
+```
+[default]
+aws_access_key_id = ACCESS_KEY_ID_HERE
+aws_secret_access_key = SECRET_ACCESS_KEY_HERE
+```
+
+To install the `aws` command from within the SDK, run:
+```
+sudo emerge --ask awscli
+```
+
+### esx
+`esx` uses `~/.config/esx.json`. This can be configured manually:
+```
+{
+    "default": {
+        "server": "server.address.goes.here",
+        "user": "user.goes.here",
+        "password": "password.goes.here"
+    }
+}
+```
+
+### gce
+`gce` uses the `~/.boto` file. This will be populated when the `gce` platform is first used.
+When the `gce` platform is first used, it will generate a link that can be used to log into your account with
+gce and get a verification code you can paste in. This will populate the `.boto` file.
+
+See [Google Cloud Platform's Documentation](https://cloud.google.com/storage/docs/boto-gsutil)
+for more information about the `.boto` file.
+
+### oci
+`oci` uses two files: `~/.oraclebmc/config` and `~/.oraclebmc/config.mantle`. The `~/oraclebmc/config` file is
+also used by the oracle `bmcs` tool. It can be populated manually:
```

Comment:
Manually is not a great solution because the key pair also needs to be generated and the public key uploaded.  We should at least mention `oci setup config` and `oci setup keys`.

Reply:
> We should at least mention `oci setup config` and `oci setup keys`

I assume `setup config` sets up the `config` file and `setup keys` sets up the key file. Did the location of where those files get created change with the `bmcs` to `oci` change?

Reply:
Looks like when they changed from `bmcs` to `oci` they moved the location to `~/.oci/`. I'll submit a PR to swap the default config location to that.

Reply:
I don't know, I'm still on `bmcs`. 😀

Reply:
https://github.com/coreos/mantle/pull/737 changes the location to `~/.oci`

Reply:
Do we have a way to install `oci` in the sdk yet?

Reply:
Nope.